src/pkcs1_5.c

```diff
@@ -244,16 +244,17 @@ int ri_manifest_sign_v1_5(struct image *image)
 {
 	struct fw_image_manifest_v1_5 *man = image->fw_image;
 
+	return
 	pkcs_v1_5_sign_man_v1_5(image, man,
 				(void *)man + MAN_CSS_MAN_SIZE_V1_5,
 				image->image_end - sizeof(*man));
-	return 0;
 }
 
 int ri_manifest_sign_v1_8(struct image *image)
 {
 	struct fw_image_manifest_v1_8 *man = image->fw_image;
 
+	return
 	pkcs_v1_5_sign_man_v1_8(image, man, (void *)man + MAN_CSS_HDR_OFFSET,
 				sizeof(struct css_header_v1_8) -
 				(MAN_RSA_KEY_MODULUS_LEN +
```

Comment:
How nicely argument list :) Is guess you added this return statement in the line before to keep this following monster as it is, but it looks quite odd (this function name is not so long, especially for 100 columns in row). Some local variable used to keep calculated size value might a good idea here, then return would easily fit before function name and be consistent with the rest of the code.

Reply:
Done, thanks for the suggestion! 2 commits now, see updated github description and commit messages